src/lightning_app/utilities/exceptions.py

```diff
@@ -41,6 +41,10 @@ class LightningWorkException(Exception):
     """Exception used to inform users of misuse with LightningWork."""
 
 
+class LightningPlatformException(Exception):
+    """Exception used to inform users of issues related to platform the LightningApp is running on."""
+
```

Comment:
Can you point us to where this is being raised?

Reply:
Directly in the lightning_launcher ([here](https://github.com/gridai/grid/pull/7789/files)) while creating a work resource. So it's not raised by framework - but for framework from platform.

Reply:
@awaelchli Do you think that's now ok? Or you mean comment like 
```
# This is raised by platform for the framework to signal platform-dependent problems.
class Light...
```

Reply:
Yes that's what I had in mind. 
I posted a suggestion how to improve the sentence.